client/layout/guided-tours/tours/checklist-domain-register-tour.js

```diff
@@ -0,0 +1,86 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React, { Fragment } from 'react';
+import Gridicon from 'gridicons';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	ButtonRow,
+	Continue,
+	makeTour,
+	Next,
+	Quit,
+	SiteLink,
+	Step,
+	Tour,
+} from 'layout/guided-tours/config-elements';
+
+export const ChecklistDomainRegisterTour = makeTour(
+	<Tour name="checklistDomainRegister" version="20180717" path="/domains/add" when={ noop }>
+		<Step
+			name="init"
+			target=".search__icon-navigation"
+			arrow="top-left"
+			placement="below"
+			style={ {
+				animationDelay: '0.7s',
+				zIndex: 1,
+			} }
+		>
+			{ ( { translate } ) => (
+				<Fragment>
+					<p>{ translate( 'Type a memorable name here to help people find your site.' ) }</p>
+					<ButtonRow>
+						<Continue target=".search__input" step="search-results" click hidden />
+						<Next step="search-results" />
+						<SiteLink isButton={ false } href="/checklist/:site">
+							{ translate( 'Return to the checklist' ) }
+						</SiteLink>
+					</ButtonRow>
+				</Fragment>
+			) }
+		</Step>
+
+		<Step name="search-results" arrow="bottom-left" placement="right">
```

Comment:
I am not sure this step is necessary if we combine the copy with the previous step.